code/datums/datumvars.dm

```diff
@@ -1351,3 +1351,9 @@
 				message_admins(msg)
 				admin_ticket_log(L, msg)
 				vv_update_display(L, Text, "[newamt]")
+		else if(href_list["copyoutfit"])
+			if(!check_rights(R_SPAWN))
+				return
+			var/mob/living/carbon/human/H = locate(href_list["copyoutfit"]) in GLOB.mob_list

```

Comment:
bit of a nit, but we have a `GLOB.carbon_list` for faster lookup, has a couple hundred fewer items.